cfgov/jinja2/v1/_includes/molecules/call-to-action.html

```diff
@@ -0,0 +1,38 @@
+{# ==========================================================================
+
+   call_to_action.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a call to action when given:
+
+   settings:             An object of used to customize the markup.
+
+   settings.text:        The text to include within slug markup.
+
+   settings.button:      Button options to create markup.
+
+   settings.button.href: A link to a page.
+                         Defaults to '/'.
+
+   settings.button.text: The text to display on button.
+
+   ========================================================================== #}
+
+{% macro render(settings={}, text='') %}
```

Comment:
Do you need to declare settings as an empty object? We should update all the others if so.